### ISSUE TITLE
More detailed installation information

### ISSUE THREAD

**Issue by Deepesh9620** [Zeptosec/ThermostoreFront#18]
Hi could you please provide more detailed instructions on how to see this up locally. What are the things required in the supabase database

**Comment by bianvigano** [comment]
you mean how to use on localhost?

**Comment by Deepesh9620** [comment]
Yes and also the database where can I setup the subabase database. I already have a cloud hosted postgresql database can I use it?

**Comment by bianvigano** [comment]
I don't know how to install supabase on localhost
but as far as I know, Supabase also uses postgresql

**Comment by Deepesh9620** [comment]
I already have an database from neon.tech can I use that?

**Comment by bianvigano** [comment]
That's actually possible, but the contents of the code will change because it was created using Supabase.

**Comment by Deepesh9620** [comment]
Is there any tutorial video on how to use this application locally (not the database)

**Comment by bianvigano** [comment]
Nothing tutorial video
